helpdesk/views/public.py

```diff
@@ -77,6 +77,10 @@ def homepage(request):
 def view_ticket(request):
     ticket_req = request.GET.get('ticket', '')
     email = request.GET.get('email', '')
+    # If there is no email address in the query string, get it from
+    # the currently logged-in user
+    if email == '':
+        email = request.user.email
```

Comment:
This has the effect of raising `AttributeError: 'AnonymousUser' object has no attribute 'email'` if a user attempts to view a ticket while not authenticated.